exporting/exporting_engine.c

```diff
@@ -4,6 +4,64 @@
 
 static struct engine *engine = NULL;
 
+void analytics_exporting_connectors (BUFFER *b)
+{
+    if (!engine)
+        return;
+
+    uint8_t count = 0;
+
+    for (struct instance *instance = engine->instance_root; instance; instance = instance->next) {
+
+        if (count) buffer_strcat(b, "|");
+
+        switch (instance->config.type) {
+            case EXPORTING_CONNECTOR_TYPE_GRAPHITE:
+                buffer_strcat(b, "Graphite");
+                break;
+            case EXPORTING_CONNECTOR_TYPE_GRAPHITE_HTTP:
+                buffer_strcat(b, "GraphiteHTTP");
+                break;
+            case EXPORTING_CONNECTOR_TYPE_JSON:
+                buffer_strcat(b, "Json");
```

Comment:
Considering that JSON means` JavaScript Object Notation`,  I suggest to use `JSON`, instead `Json`.

Reply:
Fixed!